engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/ConfigFactoryExt.scala

```diff
@@ -19,17 +19,19 @@ object ConfigFactoryExt {
     }
   }
 
-  def load(locationString: String = System.getProperty(configLocationProperty)): Config = {
+  def load(locationString: String = System.getProperty(configLocationProperty), classLoader: ClassLoader = getClass.getClassLoader): Config = {
```

Comment:
I don't like this default: `getClass.getClassLoader`. Picking correct classloader is tricky thing and someone can omit it by mistake. Maybe we should have explicitly named method for this option? Also the default shouldn't be context classloader?

Reply:
I removed default. In fact we use it only at the start of application, so "any" classloader will do - but you're right, default classloader can be error prone :)